conf.d/docker_daemon.yaml.example

```diff
@@ -51,6 +51,12 @@ instances:
     #       enable if absolutely necessary.
     # custom_cgroups: false
 
+    # Report docker container healthcheck events as service checks
+    # Note: enabling this option modifies the way in which we inspect the containers and causes
+    #       some overhead - if you run a high volume of containers we may timeout. Please only
+    #       enable if absolutely necessary.
```

Comment:
There should also be a mention that this option is available starting from Docker 1.12 and enabling it with an older version will result in an UNKNOWN state for the service check. It should be obvious but I'd rather be extra explicit here.